### PR TITLE
Add unit test for pull.request.synchronize event

### PR DESCRIPTION
**Pull request by MVrachev**
I created a pull.request.synchronize.json file for testing purposes and added a unit test in test/index.test.js

Signed-off-by: Martin Vrachev <mvrachev@vmware.com>

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -141,6 +142,40 @@ describe('Bandit-linter', () => {
       }))
     })
 
+    // This event happens when there is already opened pull request which
+    // happens across forks. Then when there is a new commit to this pull request
```
Comment (ericwb) [inline test/index.test.js]:
```suggestion
    // happens across forks. Then a new commit to this pull request,
```

File: test/index.test.js [hunk 2]
```diff
@@ -141,6 +142,40 @@ describe('Bandit-linter', () => {
...
+    // a pull_request.synchronize event will be received
```
Comment (ericwb) [inline test/index.test.js]:
```suggestion
    // pull_request.synchronize event, will be received.
```

File: test/events/pull_request.synchronize.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
+{
```
Comment (ericwb) [inline test/events/pull_request.synchronize.json]: The indentation is inconsistent, some 2 spaces, some more.

File: test/events/pull_request.synchronize.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+         }
+        },
+        "repository": {
+          "id": 54321,
+          "name": "original_repo_name",
+          "full_name": "original_repo_owner/original_repo_name",
+          "owner": {
+            "login": "original_repo_owner"
+          }
+        }
```
Comment (ericwb) [inline test/events/pull_request.synchronize.json]: note the indentation here

File: test/events/pull_request.synchronize.json [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    }
```
Comment (ericwb) [inline test/events/pull_request.synchronize.json]: note the indentation here

File: test/index.test.js (hunk 1 above)
Comment (MVrachev) [reply test/index.test.js]: I have other suggestion. I will commit it.

**Pull request merged**